onedal/version.hpp

```diff
@@ -0,0 +1,25 @@
+
+/*******************************************************************************
+* Copyright 2023 Intel Corporation
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*     http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*******************************************************************************/
+
+#pragma once
+
+#include "services/library_version_info.h"
+
+#define MAJOR_VERSION  __INTEL_DAAL__
+#define MINOR_VERSION  __INTEL_DAAL_MINOR__
+#define UPDATE_VERSION __INTEL_DAAL_UPDATE__
+#define ONEDAL_VERSION MAJOR_VERSION * 10000 + MINOR_VERSION * 100 + UPDATE_VERSION
```

Comment:
It is equivalent of `INTEL_DAAL_VERSION` from "services/library_version_info.h". Replace it with original

Reply:
Fixed